mcs/class/corlib/System/Environment.cs

```diff
@@ -48,16 +48,11 @@ namespace System {
 	public static partial class Environment {
 
 		/*
-		 * This is the version number of the corlib-runtime interface. When
-		 * making changes to this interface (by changing the layout
-		 * of classes the runtime knows about, changing icall signature or
-		 * semantics etc), increment this variable. Also increment the
-		 * pair of this variable in the runtime in metadata/appdomain.c.
-		 * Changes which are already detected at runtime, like the addition
-		 * of icalls, do not require an increment.
+		 * This is the version number of the corlib-runtime interface.
```

Comment:
Hi Zoltan,

We decided to lose the shackles of decisions made in the 2000-2004 timeframe and give the corlib version a meaning more useful for our current reality.

Starting from this PR, the corlib version should be fully usable by monolite, so it must reflect changes that would break monolite as well.